lib/ts/recipe/thirdparty/utils.ts

```diff
@@ -34,9 +34,9 @@ export function validateAndNormaliseUserInput(appInfo: NormalisedAppinfo, config
 
 function validateAndNormaliseSignInAndUpConfig(
     _: NormalisedAppinfo,
-    config: TypeInputSignInAndUp
+    config: TypeInputSignInAndUp | undefined
 ): TypeNormalisedInputSignInAndUp {
-    let providers = config.providers;
+    let providers: ProviderInput[] = config?.providers || [];
```

Comment:
i do not like implicit checks.. please be explicit - leads to lesser bugs

Reply:
Done